net-wireless/nitroshare/nitroshare-0.4.0~git.recipe

```diff
@@ -10,21 +10,29 @@ DESCRIPTION="Features
 * Local HTTP API
 * Completely free and open-source"
 HOMEPAGE="https://github.com/nitroshare/nitroshare-desktop"
-COPYRIGHT="2017 Nathan Osman"
+COPYRIGHT="2015-2019 Nathan Osman"
 LICENSE="MIT"
 REVISION="1"
-SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
-CHECKSUM_SHA256="29874e5909c29211a3c9e13f8c0f49b901ec2996e5d60d80af80d2fb80c3d7ec"
-SOURCE_FILENAME="nitroshare-$portVersion.tar.gz"
-SOURCE_DIR="nitroshare-desktop-$portVersion"
+srcGitRev="a2fc0eae88fb2ea8a13f02e052a0af47b8532a2f"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="5aee1aa91f800068d5ee3428a9d900a768ba3fea93e329c52b5e0cd12e3852b6"
+SOURCE_FILENAME="nitroshare-$srcGitRev.tar.gz"
+SOURCE_DIR="nitroshare-desktop-$srcGitRev"
 PATCHES="nitroshare-$portVersion.patchset"
 
-ARCHITECTURES="all !x86_gcc2 ?x86"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="?x86"
+ADDITIONAL_FILES="nitroshare.rdef"
```

Comment:
Additional_files needs to be moved before architectures: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering